examples/11-using-facts-in-events.js

```diff
@@ -0,0 +1,143 @@
+"use strict";
```

Comment:
just a friendly reminder to fix the linting errors 🙂

Reply:
oops.